app/server/docker/entrypoint.sh

```diff
@@ -7,4 +7,4 @@ while [ ! -e $PATH_CONFIG ]; do
   sleep 1
 done
 
-node dist/index.js --config /app/src/resources
+ts-node --transpile-only src/index.ts --config src/resources
```

Comment:
Running direct from the js started throwing some fairly fundamental errors (struggling with import types). I think running ts-node should be ok with transpile-only to improve startup time.

Reply:
is this just to be done so we can run it locally? because this works already in prod? sorry or im mistaken?